diff --git a/lib/vue-sonner.d.ts b/lib/vue-sonner.d.ts
index 73ce85452c54ef186eece1bd4c2521b513fe29ae..f893242c6b03e8ada974736b71bca1e5b8595bf7 100644
--- a/lib/vue-sonner.d.ts
+++ b/lib/vue-sonner.d.ts
@@ -63,7 +63,7 @@ export declare const toast: ((message: string | Component, data?: ExternalToast)
     loading: (message: string | Component, data?: ExternalToast) => string | number;
 };
 
-declare interface ToastClasses {
+export declare interface ToastClasses {
     toast?: string;
     title?: string;
     description?: string;
@@ -179,7 +179,7 @@ declare interface ToastOptions {
     classes?: ToastClasses;
 }
 
-declare type ToastT<T extends Component = Component> = {
+export declare type ToastT<T extends Component = Component> = {
     id: number | string;
     title?: string | Component;
     type?: ToastTypes;
